openapi/multi-yaml/paths/account@{account_id}@payout_holds.yaml

```diff
@@ -0,0 +1,61 @@
+get:
+  summary: List Payout Holds
+  description: Retrieve a paginated list of payout holds for an account.
+  operationId: ListPayoutHolds
+  tags:
+    - Payout Holds
+  parameters:
+    - $ref: ../components/parameters/account-id-path.yaml
+    - $ref: ../components/parameters/authorization-header.yaml
+    - $ref: ../components/parameters/pagination-after.yaml
+    - $ref: ../components/parameters/created-before.yaml
+    - $ref: ../components/parameters/created-after.yaml
+    - $ref: ../components/parameters/start-date-before.yaml
+    - $ref: ../components/parameters/start-date-after.yaml
+    - $ref: ../components/parameters/end-date-before.yaml
+    - $ref: ../components/parameters/end-date-after.yaml
+  responses:
+    '200':
+      description: Successfully list payout holds
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: ../components/schemas/Envelope-list.yaml
+              - properties:
+                  type:
+                    example: array
+                  data:
+                    items:
+                      $ref: ../components/schemas/PayoutHold.yaml
+
+post:
+  summary: Create Payout Hold
+  description: Create a new payout hold for a seller account.
+  operationId: CreatePayoutHold
+  tags:
+    - Payout Holds
+  parameters:
+    - $ref: ../components/parameters/account-id-path.yaml
```

Comment:
Same here - see my earlier question about why the account_id is needed in the path.